src/Arcus.Testing.Logging/SpyLogger.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using GuardNet;
+using Microsoft.Extensions.Logging;
+
+namespace Arcus.Testing.Logging
+{
+    /// <summary>
+    /// Spy (stub) <see cref="ILogger"/> implementation to track the logged messages.
+    /// </summary>
+    public class SpyLogger : ILogger
```

Comment:
`Spy`, `Stub` or `InMemory`?

Reply:
Stub is maybe to generic for this. It's definitly used for 'spying' during testing, but it's indeed an 'in-memory' approach to do so. Maybe indeed changing to `InMemoryLogger` bc I don't know how well people are aware of the different testing terminology.

Reply:
Let's change it and we are good to go!

Reply:
If it's called SpyLogger I wouldn't find it without reading the docs for sure.